weblate/addons/resx.py

```diff
@@ -67,7 +67,7 @@ def update_resx(self, index, translation, storage, changes):
 
     @staticmethod
     def find_changes(index, storage):
-        """Find changed units in storage"""
+        """Find changed string IDs in upstream repository"""
```

Comment:
In the code, we still call these units. Also it's not a repository, it's a file storage.

Reply:
The difference is lost on me here. Does it just find them, or something more? How about "Find changed string IDs upstream"?

Reply:
Unit is an object representation of what we call string in the interface. Mostly for historical reasons it's called differently in the code (actually it matches translate-toolkit terminology here).